### PR TITLE
MGMT-3205 Enable/Disable OCS to the cluster definition

### PR DESCRIPTION
**Pull request by priyanka19-98**
The PR is required to test integration of OCS with assisted-service: 
Ref PR: https://github.com/openshift/assisted-service/pull/637
Signed off by: Priyanka Jiandani <pjiandan@redhat.com>

cc: @rollandf  @nbalacha

### CODE REVIEW

File: discovery-infra/test_infra/consts.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 INSTALLING_IN_PROGRESS_TIMEOUT = 60 * 10
 VALIDATION_TIMEOUT = 6 * 60
 NTP_VALIDATION_TIMEOUT = 10 * 60
+OCS_VALIDATION_TIMEOUT = 10 * 60
```
Comment (nbalacha) [inline discovery-infra/test_infra/consts.py:15]: What is this timeout used for? Only for the validation or the complete install?

Comment (priyanka19-98) [reply discovery-infra/test_infra/consts.py:15]: only for validations, where we need to allow hosts to be discovered after changing cps, ram

**Review by gamli75 — APPROVED** [review]

File: discovery-infra/tests/conftest.py [hunk 2]
```diff
@@ -49,7 +49,10 @@ def is_qe_env():
                  "service_cidr": utils.get_env('SERVICE_CIDR', '172.30.0.0/16'),
                  "cluster_cidr": utils.get_env('CLUSTER_CIDR', '10.128.0.0/14'),
                  "host_prefix": int(utils.get_env('HOST_PREFIX', '23')),
-                 "iso_image_type": utils.get_env('ISO_IMAGE_TYPE', consts.ImageType.FULL_ISO)}
+                 "iso_image_type": utils.get_env('ISO_IMAGE_TYPE', consts.ImageType.FULL_ISO),
+                 "worker_vcpu": 4,
```
Comment (tsorya) [inline discovery-infra/tests/conftest.py:53]: It must not be hardcoded values add an option to read them from os env

Comment (tsorya) [reply discovery-infra/tests/conftest.py:53]: and add 4 to consts

Comment (priyanka19-98) [reply discovery-infra/tests/conftest.py]: fixed

**Pull request merged**